server/server.js

```diff
@@ -13,4 +13,12 @@ useMorganMiddleware(app);
 app.use(express.static(assets));
 app.use('/test', router);
 
+app.get('/*', (req, res) => {
+  res.sendFile(path.join(__dirname, '../client/'), (err) => {
```

Comment:
use `assets` declaration or remove it. Repeated code otherwise.

Reply:
Addresses the other comment too^

I can't get react-router to handle the catch-all fallback unless both exist.

I've tried the following:

Removing the express.static code, leaving the catch-all block, and it won't render the map at all.

Replacing the argument within `sendFile` with `assets`.  The browser shows the following output: `{"code":"EISDIR"}`.

Not sure exactly what to do here, but if you have a working solution, I could use some help.